spec/quickstart/quickstart-cli.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Created by krzysztof on 01.02.16.
+ */
+
+'use strict';
+
+var logger = require('log4js').getLogger('CLI');
+var SSH = require('simple-ssh');
+var fs = require('fs');
+
+process.chdir('build/vagrant-quickstart');
+
+describe('CLI test', function () {
+
+    var deploymentName = 'nodecellar';
```

Comment:
Save strings in the config 'defaults.json' https://github.com/cloudify-cosmo/cloudify-ui-selenium-tests-nodejs/blob/master/src/components/config/defaults.json.